modules/ROOT/pages/indexes/indexes-for-full-text-search.adoc

```diff
@@ -0,0 +1,394 @@
+
+In the following example, the previously created `communications` full-text index is deleted from the database:
+
+.Drop a full-text index
+[source, cypher]
+----
+DROP INDEX communications
+----
+
+[[full--text-index-procedures]]
+== List of full-text index procedures
+
+Full-text indexes are managed through commands and used through built-in procedures, see link:{neo4j-docs-base-uri}/operations-manual/{page-version}/reference/procedures[Operations Manual -> Procedures] for a complete reference.
+
+The procedures for full-text indexes are listed in the table below:
+
+[options="header"]
+|===
+| Usage | Procedure/Command | Description
+
+| List available analyzers.
+| https://neo4j.com/docs/operations-manual/current/reference/procedures/#procedure_db_index_fulltext_listavailableanalyzers[`db.index.fulltext.listAvailableAnalyzers`]
+| List the available analyzers that the full-text indexes can be configured with.
+
+| Use full-text node index.
+| https://neo4j.com/docs/operations-manual/current/reference/procedures/#procedure_db_index_fulltext_querynodes[`db.index.fulltext.queryNodes`]
+| Query the given full-text index. Returns the matching nodes and their Lucene query score, ordered by score.
+
+| Use full-text relationship index.
+| https://neo4j.com/docs/operations-manual/current/reference/procedures/#procedure_db_index_fulltext_queryrelationships[`db.index.fulltext.queryRelationships`]
+| Query the given full-text index. Returns the matching relationships and their Lucene query score, ordered by score.
+
+| Eventually consistent indexes.
+| https://neo4j.com/docs/operations-manual/current/reference/procedures/#procedure_db_index_fulltext_awaiteventuallyconsistentindexrefresh[`db.index.fulltext.awaitEventuallyConsistentIndexRefresh`]
+| Wait for the updates from recently committed transactions to be applied to any eventually-consistent full-text indexes.
+
+|===
+
+[[summary]]
+== Summary
+
+The following is true for full-text indexes:
+
+* Full-text indexes support the indexing of both nodes and relationships.
+* Full-text indexes include only property values of types `STRING` or `LIST<STRING>`.
+* Full-text indexes are accessed via Cypher procedures.
+* Full-text indexes can return the _score_ for each result from a query.
+* Full-text indexes support configuring custom analyzers, including analyzers that are not included with Lucene itself.
+* Full-text indexes can be queried using the Lucene query language.
+* Full-text indexes are kept up to date automatically, as nodes and relationships are added, removed, and modified.
+* Full-text indexes will automatically populate newly created indexes with the existing data in a store.
+* Full-text indexes can be checked by the consistency checker, and they can be rebuilt if there is a problem with them.
+* Full-text indexes are a projection of the store, and can only index nodes and relationships by the contents of their properties.
```

Comment:
What does this point mean, and what extra useful info does it try to convey?

Reply:
Nothing, I think - the same is true for all search-performance indexes. Let's get rid of it.